build.sbt

```diff
@@ -1,18 +1,22 @@
 ThisBuild / tlBaseVersion                         := "0.34"
 ThisBuild / tlCiReleaseBranches                   := Seq("master")
 ThisBuild / githubWorkflowEnv += "MUNIT_FLAKY_OK" -> "true"
-ThisBuild / scalacOptions += "-Xsource:3"
 
 Global / concurrentRestrictions += Tags.limit(Tags.Compile, 1)
 
+ThisBuild / scalaVersion := "3.1.2"
+ThisBuild / crossScalaVersions := Seq("3.1.2")
+
+ThisBuild / resolvers += Resolver.sonatypeRepo("snapshots")
+
 lazy val attoVersion           = "0.9.5"
 lazy val catsVersion           = "2.7.0"
 lazy val kindProjectorVersion  = "0.13.2"
 lazy val monocleVersion        = "3.1.0"
 lazy val scalaJavaTimeVersion  = "2.3.0"
 lazy val geminiLocalesVersion  = "0.7.0"
 lazy val jtsVersion            = "0.2.2"
-lazy val coulombVersion        = "0.5.8"
+lazy val coulombVersion        = "0.6-3cc9f7f-SNAPSHOT"
```

Comment:
```suggestion
lazy val coulombVersion        = "0.6-7c69de6-SNAPSHOT"
```

Reply:
@erikerlandson thanks for the quick fix, it's working now! :)

Reply:
:tada:   thanks for exercising the code!